jekyll/_cci2/caching.md

```diff
@@ -59,13 +60,16 @@ As in CircleCI 1.0, it is possible and oftentimes beneficial to cache your git r
           paths:
             - ".git"
 ```
+
 {% endraw %}
 
-In this example, `restore_cache` looks for a cache-hit from the current git revision, then for a hit from the current branch, then, most broadly, for any cache-hit at all, regardless of branch or revision. If your source code changes frequently, or changes significantly across branches, you may want to remove the third `restore_cache` key, or even both the second *and* third keys, resulting in a more granular source cache that only looks for matches within the current branch and git revision.
+In this example, `restore_cache` looks for a cache hit from the current git revision, then for a hit from the current branch, and finally for any cache hit, regardless of branch or revision. When CircleCI encounters a list of `keys`, the cache will be restored from the first match. If there are multiple matches, the **most recent match** will be used.
+
+If your source code changes frequently, we recommend using fewer, more specific keys. This produces a more granular source cache that will update more often as the current branch and git revision change.
```

Comment:
Just a style nit, which is to change 'we recommend' to 'consider'. Otherwise this looks great, thanks for improving this!